src/test/java/com/puppycrawl/tools/checkstyle/filters/IntMatchFilterElementTest.java

```diff
@@ -51,7 +48,8 @@ public void testEqualsAndHashCode() {
     @Test
     public void testToString() {
         final IntFilterElement filter = new IntMatchFilterElement(6);
-        assertEquals("IntMatchFilterElement[6]", filter.toString(), "Invalid toString result");
+        assertWithMessage("Invalid toString result").that(filter.toString())
+                .isEqualTo("IntMatchFilterElement[6]");
```

Comment:
it would be awesome if you format it like:
```
assertWithMessage("Invalid toString result")
                .that(filter.toString())
                .isEqualTo("IntMatchFilterElement[6]");
```

to focus quickly on `.that` - as the most critical part of assert.

Reply:
https://github.com/checkstyle/checkstyle/issues/11087